cpp/tests/interop/to_arrow_test.cpp

```diff
@@ -112,8 +112,8 @@ std::pair<std::unique_ptr<cudf::table>, std::shared_ptr<arrow::Table>> get_table
     cudf::test::strings_column_wrapper(string_data.begin(), string_data.end(), validity.begin())
       .release();
   vector_of_columns cols;
-  cols.push_back(move(int_column));
-  cols.push_back(move(str_column));
+  cols.push_back(std::move(int_column));
+  cols.push_back(std::move(str_column));
```

Comment:
how does  just`move` work here? 
any using namespace std? or type alias for `std::move`?

Reply:
I'm guessing that something upstream of us must be doing it. I couldn't find it in any of the files we include, although I may have missed something.